SlowCheetah.VisualStudio/SlowCheetahPackage.cs

```diff
@@ -100,6 +107,38 @@ public string GetVsInstallDirectory()
             return installDirectory;
         }
 
+        /// <inheritdoc/>
+        public int UpdateSolution_Begin(ref int pfCancelUpdate)
+        {
+            return VSConstants.E_NOTIMPL;
```

Comment:
Don't return E_NOTIMPL if you are intentionally doing a no-op, return `VSConstants.S_OK`